SPECS-EXTENDED/perl-Net-LibIDN2/perl-Net-LibIDN2.spec

```diff
@@ -1,12 +1,13 @@
 Vendor:         Microsoft Corporation
 Distribution:   Azure Linux
 Name:           perl-Net-LibIDN2
-Version:        1.01
-Release:        3%{?dist}
+Version:        1.02
+Release:        1%{?dist}
```

Comment:
Increase the release number to 9.

Reply:
I had a chat with the author of this PR. Since it is just an upgrade no need to increase the release number.